base/setup/lib/utils/partlist.c

```diff
@@ -2782,36 +2782,110 @@ GetNextUnpartitionedEntry(
     return NULL;
 }
 
+ERROR_NUMBER
+PartitionCreationChecks(
+    _In_ PPARTENTRY PartEntry)
+{
+    PDISKENTRY DiskEntry = PartEntry->DiskEntry;
+
+    if (DiskEntry->DiskStyle == PARTITION_STYLE_GPT)
+    {
+        DPRINT1("GPT-partitioned disk detected, not currently supported by SETUP!\n");
```

Comment:
```suggestion
        DPRINT1("GPT-partitioned disk detected, not currently supported by SETUP\n");
```

Reply:
Don't care about this, because:
1. This is from existed but moved code;
2. Will disappear once GPT support is added (scheduled to be part of 1st-stage GUI setup program or following).